demo/mxcube-web/server.yaml

```
@@ -45,6 +45,9 @@ server:
   CSP_REPORT_ONLY: false
   CSP_REPORT_URI: "/mxcube/api/v0.1/csp/report"
 
+  GRAYLOG_HOST: "graylog-dau.esrf.fr"
+  GRAYLOG_PORT: 12210
```

```suggestion
  GRAYLOG_HOST: ""
  GRAYLOG_PORT: 0
```
I don't think that address of some internal ESRF service is a good default value :smile_cat: 
I think the default value should be falsy, as the Graylog integration should be optional.
Having these keys in `server.yaml` is of course a good idea, as it makes it self-documenting. I believe it would be also nice to mention it [there](https://github.com/mxcube/mxcubeweb/blob/develop/docs/source/config/server.rst).